src/datepicker/datepicker.ts

```diff
@@ -84,7 +84,7 @@ export interface NgbDatepickerNavigateEvent {
             [showWeekdays]="showWeekdays"
             [showWeekNumbers]="showWeekNumbers"
             [disabled]="disabled"
-            [outsideDays]="displayMonths === 1 ? outsideDays : 'hidden'"
+            [outsideDays]="outsideDays || displayMonths === 1 ? 'visible' : 'hidden'"
```

Comment:
I think with operator precedence you need some parens here:

```
[outsideDays]="outsideDays || (displayMonths === 1 ? 'visible' : 'hidden')"
```

Otherwise you'll be modifying the value of the month view's `outsideDays` to always be "visible" any time it is explicitly set on the outer datepicker.